code/_core/datum/damagetype/ranged/bullet/rifle_223.dm

```diff
@@ -63,12 +63,14 @@
 
 	//The base attack damage of the weapon. It's a flat value, unaffected by any skills or attributes.
 	attack_damage_base = list(
-		PIERCE = DAMAGE_SWORD
+		PIERCE = DAMAGE_SWORD,
+		BIO = DAMAGE_SWORD*0.5
 	)
 
 	//How much armor to penetrate. It basically removes the percentage of the armor using these values.
 	attack_damage_penetration = list(
-		PIERCE = AP_SWORD
+		PIERCE = AP_SWORD,
+		BIO = AP_SWORD
```

Comment:
Same logic above should apply to armor penetration too unless it is AP round.

Reply:
Yes.